oaknational/Oak-Web-Application
# Feat/search dropdown tracking events LESQ-537

## Description

Music year: 1989

- adds unit title to lesson search hit
- refactors tracking for onclick search hits - searchResultOpened
- adds tracking for on toggle open (expanded search results) - searchResultsExpanded
- adds tests for both

## Issue(s)

Fixes #

## How to test

Go to search and check tracking events.
searchResultOpened
searchResultsExpanded
https://www.avo.app/schemas/5PhajbVijwhXVKIJtGMT/branches/Rx5PGc6-w/events/e/eP2lOQQj8C/e/_TD-gtSgIj

These should also be coming through correctly in posthog.


## Checklist
